GPT/gpt-cursorless.talon

```diff
@@ -9,3 +9,10 @@ model <user.modelPrompt> <user.cursorless_target> [<user.cursorless_destination>
     result = user.gpt_apply_prompt(user.modelPrompt, text)
     default_destination = user.cursorless_create_destination(cursorless_target)
     user.cursorless_insert(cursorless_destination or default_destination, result)
+
+model bring <user.cursorless_target> to <user.cursorless_target>$:
```

Comment:
Almost wonder if this should be named `model merge`. But I understand wanting to keep cursorless syntax too.